tests/duplicates.robot

```diff
@@ -0,0 +1,50 @@
+*** Settings ***
+Documentation  Suite for duplicated test cases and keywords
+Library   PythonLibrary  1
+Library   PythonLibrary  1
+Resource  resource.robot
+Library   PythonLibrary  2
+Library   PythonLibrary
+Resource  resource2.robot
+Resource  resource.robot
```

Comment:
What about Metadata?

Reply:
I was still thinking how I should handle it (for example there could be multiline metadata that is duplicate of metadata with one long line). I will update this PR with metadata checks